### PR TITLE
Create 1-legendary.md

### PR DESCRIPTION
**Pull request by mrjones2014**
Requesting community feedback for `legendary.nvim`

### CODE REVIEW

**Review by phaazon — APPROVED** [review]
Thanks!

File: contents/2022/Nov/07/1-need-help/1-legendary.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<h3 id="help-your-plugin.nvim">
```
Comment (haolian9) [inline contents/2022/Nov/07/1-need-help/1-legendary.md:1]: maybe h3.id and a.href need to be updated.

File: contents/2022/Nov/07/1-need-help/1-legendary.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  </a>
+</h3>
+
+> 🗺️ A legend for your keymaps, commands, and autocmds, with which-key.nvim integration
+
+`legendary.nvim` will be introducing semantic versioning via git tags in order to safely allow for a large refactor/rewrite.
+Feedback is requested to help make decisions during the rewrite.
+If you are a `legendary.nvim` user, please provide feedback via this [GitHub discussion](https://github.com/mrjones2014/legendary.nvim/discussions/188).
+
+- [GitHub](https://github.com/mrjones2014/legendary.nvim/discussions/188)
```
Comment (haolian9) [inline contents/2022/Nov/07/1-need-help/1-legendary.md:18]: typo?

Comment (mrjones2014) [reply contents/2022/Nov/07/1-need-help/1-legendary.md:18]: Thanks, I filed a new PR to fix these.

**Pull request merged**